src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -763,24 +765,29 @@ def phot_centroid(data, meta):
     csx_rms = np.sqrt(np.nanmean((csx - np.nanmedian(csx)) ** 2))
     csy = data.centroid_sy.values
     csy_rms = np.sqrt(np.nanmean((csy - np.nanmedian(csy)) ** 2))
+    flat_line = np.zeros(len(data.centroid_x))
 
     ax[0].plot(data.time, data.centroid_x-np.nanmean(data.centroid_x),
                label=r'$\sigma$x = {0:.4f} pxls'.format(cx_rms))
+    ax[0].plot(data.time, flat_line, linestyle=':', color='r')
```

Comment:
For these flat lines, can you instead use `ax[0].axhline` instead so that it spans the full x-axis